docker-compose_gpu.yaml

```diff
@@ -9,23 +25,16 @@ services:
       - chromadb_data:/chroma/.chroma/index
 
   chatollama:
+    container_name: chatollama
     environment:
       - CHROMADB_URL=http://chromadb:8000
       - DATABASE_URL=file:/app/sqlite/chatollama.sqlite
-    image: 0001coder/chatollama:latest
+    image: chatollama:vgh4
```

Comment:
Let's stick `0001coder/chatollama:latest` to this image attribute, and keep the `pull_policy` as `always` below. It makes regular users keep up with the version update easier, so a restart should be enough for pulling the latest version.